src/components/organisms/ConceptPicker.tsx

```diff
@@ -64,20 +66,7 @@ const onConceptChange = (router: NextRouter, concept: TConcept) => {
   router.push({ query: query }, undefined, { shallow: true });
 };
 
-const getSelectedConcepts = (concepts: TConcept[], router: NextRouter) => {
```

Comment:
Removed as this function wasn't being called anywhere.

Reply:
Yeh I left it in case we ever wanted to use the logic of showing the selected concepts first. Conor pointed out that we will already be showing applied filters in two other places so probably not needed.

Reply:
Hoarder-mentality 😆